fabric/src/main/resources/fabric.mod.json

```diff
@@ -11,12 +12,17 @@
   },
   "contact": {
     "homepage": "https://wynntils.com",
+    "issues": "https://github.com/Wynntils/Wynntils/issues",
     "sources": "https://github.com/Wynntils/Wynntils"
   },
   "custom": {
     "modmenu": {
       "links": {
-        "modmenu.discord": "https://discord.gg/wynntils-mod"
+        "modmenu.crowdin": "https://crowdin.com/project/wynntils-mod",
+        "modmenu.discord": "https://discord.gg/wynntils-mod",
```

Comment:
We should really use the perma invite code, in case we lose this or switch to discord.gg/wynntils again.
https://discord.gg/SZuNem8